pingcap/tidb
# telemetry: fix the bug of annoying log caused by telemetry

### What problem does this PR solve?

Issue Number: close #25760  close #25785 
fix the bug of sql staticis & slow query staticis

Problem Summary:
annoying log caused by telemetry, TiDB reports 'querySQLMetric got error' every minute.

### What is changed and how it works?

What's Changed:

How it Works:

### Related changes

- PR to update `pingcap/docs`/`pingcap/docs-cn`:
- Need to cherry-pick to the release branch

### Check List 

Tests 

- Unit test
- Integration test
- Manual test (add detailed scripts or steps below)
use `./bin/tidb-server` to set up TiDB and see the log.


Side effects

- Performance regression
    - Consumes more CPU
    - Consumes more MEM
- Breaking backward compatibility

### Release note 

- fix the bug of annoying logs caused by telemetry
